### PR TITLE
Update wrong header's props in Commerce.js

### PR DESCRIPTION
**Pull request by jaepass**
- Updated wrong headers
With many apologies for this update again! 🙏

### CODE REVIEW

File: src/technologies.json [hunk 1]
```diff
@@ -3101,11 +3101,12 @@
       ],
       "description": "Commerce.js is an API-first eCommerce platform for developers & businesses.",
       "headers": {
-        "X-Powered-By": "Commerce.js"
-      },
+         "X-Support":  "support@commercejs\\.com",
+         "Chec-Version": ".*"
```
Comment (AliasIO) [inline src/technologies.json:3105]: Can this be used to capture the version number? E.g. `"Chec-Version": "(\\d+)"\\;version=\\1`

File: src/technologies.json [hunk 2]
```diff
@@ -3101,11 +3101,12 @@
...
+       },
       "icon": "commercejs.png",
       "scripts": [
-        "^https?:/cdn\\.chec\\.io/v2/commerce\\.js/",
+        "^https?:/cdn\\.chec\\.io/v\\d+/commerce\\.js",
```
Comment (AliasIO) [inline src/technologies.json:3109]: Or this? E.g. `"^https?:/cdn\\.chec\\.io/v(\\d+)/commerce\\.js\\;version=\\1`

File: src/technologies.json (hunk 1 above)
Comment (jaepass) [reply src/technologies.json:3105]: hey yes thanks for the suggestion, that might be ideal. But do do have version 1 and 2 in the wild

Comment (ScopeyNZ) [reply src/technologies.json:3105]: The `Chec-Version` header is actually a date version that our API provides, not our SDK version. There's two versions in play here, I'm not sure if we can differentiate them in wappalyzer. I think the SDK version is more appropriate.

File: src/technologies.json (hunk 2 above)
Comment (jaepass) [reply src/technologies.json]: thanks a ton @AliasIO for the suggestion, i've just pushed it up! 🙏🙏🙏

**Comment by jaepass** [comment]
Let me know if all looks good thanks again!

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**